src/mixins/filing-template-mixin.ts

```diff
@@ -755,7 +776,7 @@ export default class FilingTemplateMixin extends DateMixin {
     this.setBusinessInformation({
       ...entitySnapshot.businessInfo,
       ...filing.business,
-      ...filing.alteration.business
+      ...filing.alteration?.business
```

Comment:
Why is this needed? Isn't the alteration object always present?

Reply:
I may have added it in when I wasn't saving the payload correctly. I'll try to remove it

Reply:
Removed, thanks.